src/control/server/privileges.go

```diff
@@ -39,14 +43,20 @@ func getUID(ext External, userName string) (*user.User, int64, error) {
 
 	uid, err := strconv.ParseInt(usr.Uid, 10, 32)
 	if err != nil {
-		return nil, -1, errors.WithMessage(err, "parsing uid to int")
+		return nil, -1, errors.Wrap(err, "parsing uid to int")
 	}
 
 	return usr, uid, nil
 }
 
-func getGID(ext External, usr *user.User, groupName string) (int64, error) {
+// getGID returns group name and id, either of the supplied group if user
+// belonga to group or of usr otherwise.
+func getGID(
+	ext External, usr *user.User, groupName string) (
+	grpName string, gid int64, err error) {
```

Comment:
Same comments on the line-breaking style...

Reply:
I've seen line breaking performed like this in various style guides, I tend to prefer line breaks after opening parenthesis. is there another way you would prefer to do this?